src/python/bcc/__init__.py

```diff
@@ -62,13 +62,47 @@ def resolve_name(self, name):
             return -1
         return addr.value
 
+class PerfType:
+    # From perf_type_id in uapi/linux/perf_event.h
+    HARDWARE = 0
+    SOFTWARE = 1
```

Comment:
According to https://github.com/torvalds/linux/commit/0515e5999a466dfe6e1924f460da599bb6821487 I think so. @4ast can confirm